falco.yaml

```diff
@@ -134,6 +134,10 @@
 # By arranging the order of files and rules thoughtfully, you can ensure that
 # desired customizations and rule behaviors are prioritized and applied as
 # intended.
+# 
+# With Falco 0.36 and beyond, it's now possible to apply multiple rules that match 
+# the same event type, eliminating concerns about rule prioritization based on the 
+# "first match wins" principle. Read more under the `rule_matching` configuration.
```

Comment:
thank you for this, I would just highlight that this has a cost in terms of perf because you have to loop on all rules every time alsp when there is a match...it should be negligible considering that most of the time no rule will match but if we want to be precise there is a penalty :)

Reply:
Yes, nice catch, we shall add this. Is it ok that I added it twice? I believe redundancy could help here.